Header header
GoalID goal_id
robot_msgs/PoseStamped result
